.github/workflows/comment_bot.yml

```diff
@@ -12,6 +12,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+          submodules: true
```

Comment:
The bot doesn't need access to the submodules

Reply:
Fixed : f406515